types/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// TypeScript Version: 3.2
+export {};
+declare global {
```

Comment:
This is needed to augment Jest expect definition so that new methods are recognized by Typescript.

Reply:
this should allow intellisense when adding expectations to tests? if so it doesn't work for me vscode. not sure if this is an issue on my end.

Reply:
Yes, exactly. Works perfectly with WebStorm when consuming as external dependency (as then this file is auto-imported due to reference in package.json).
When consumed from within the project itself, make sure to import this file directly.
Are you trying it in a TS project? Not sure if VSCode supports inferring intellisense for non-TS projects from TS types.

Reply:
Actually, IntelliSense is not even the most important part. When in TypeScript project, tests that are using new assertions wouldn't even compile without this  (due to missing methods on jest `expect`), unless you put `// @ts-ignore` all over the place, which make the feature pretty much unusable in such environment.

Reply:
Cool!

Weird it's not working for me, though.

I can accept the PR like this, but would it be possible to add this also for chai and should?

Reply:
Appreciate it!
Of course, I can also fully type the configs and validators. Can it happen in a separate PR after this one is merged, though?

Reply:
Sure, approving